Java_fibonacci.java

```diff
@@ -0,0 +1,25 @@
+package milestone6;
+
+import java.util.Scanner;
+
+public class Java_fibonacci {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		 Scanner sc=new Scanner(System.in);
+         System.out.println("Enter a number of members in series: ");
+         int n=sc.nextInt();
+         int count=0;
+         int a=1,b=1;
```

Comment:
I'm encountering the following error after executing the program. Try changing them and add a new line after printing each fibonacci term.

Enter a number of members in series: 100
1 1 2 4 8 16 32 64 128 256 512 1024 2048 4096 8192 16384 32768 65536 131072 262144 524288 1048576 2097152 4194304 8388608 16777216 33554432 67108864 134217728 268435456 536870912 1073741824 -2147483648 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 

Happy Coding!!
Happy Contributing!!